### PR TITLE
feat: self user devices (AR-2215)

### PR DESCRIPTION
**Pull request by Garzas**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2215" title="AR-2215" target="_blank">StoryAR-2215</a>  View Own List of Devices
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

A screen of self-user devices opened from the Settings panel

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1067/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4592569)(4592569)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4592569)(4592569)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1067/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4592924)(4592924)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4592924)(4592924)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/settings/devices/DevicesScreen.kt [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                            item {
+                                FolderHeader(
+                                    name = context.getString(R.string.current_device_label).uppercase(),
+                                    modifier = Modifier
+                                        .fillMaxWidth()
+                                        .background(color = headerColor)
+                                )
+                            }
+                            item { DeviceItem(currentDevice, placeholder = false) }
+                            item { Divider() }
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/ui/settings/devices/DevicesScreen.kt]: Don't make dividers as separate list items, here's the Google's recommendation:

> You do not want dividers to change scrolling indices, as they shouldn’t be considered independent elements. Also, performance will not be affected as dividers are small. A divider will likely need to be visible when the item before it is visible, so they can be part of the previous item

File: app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt [hunk 2]
```diff
@@ -165,6 +167,11 @@ enum class NavigationItem(
         content = { AppSettingsScreen() },
     ),
 
+    Devices(
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt]: I think it would be better to give it a more precise name like `SelfDevices`

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1067/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4600129)(4600129)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4600129)(4600129)-dev-release.apk)

**Review by saleniuk — APPROVED** [review]

**Pull request merged**